velox/functions/lib/window/tests/WindowTestBase.cpp

```diff
@@ -112,10 +186,19 @@ void WindowTestBase::testWindowFunction(
   if (createTable) {
     createDuckDbTable(input);
   }
+
+  auto flag = true;
```

Comment:
I feel the current logic makes it predictable the order in which input queries are mapped to the window algorithm.

It might be better to do:
i) Add an enum which specifies windowStyle to this function. The options are SORT, STREAMING, RANDOM. The default windowStyle should be RANDOM.
ii) If SORT or STREAMING then use the specified buildWindow function
iii) If RANDOM then generate a random boolean and use SORT or STREAMING based on the result.

This logic will make the test more robust and give more flexible testing API to the user.

Reply:
Updated.